controllers/operatorpolicy_status.go

```diff
@@ -600,13 +680,33 @@ func buildCSVCond(csv *operatorv1alpha1.ClusterServiceVersion) metav1.Condition
 	}
 }
 
+// noCSVCond is a NonCompliant condition with Reason 'RelevantCSVNotFound'
 var noCSVCond = metav1.Condition{
 	Type:    csvConditionType,
 	Status:  metav1.ConditionFalse,
```

Comment:
Maybe I missed something, but wouldn't this imply that the reason "RelevantCSVNotFound" is false here given the status of the condition?

Reply:
Conditions and reasons can be confusing, and that's not limited to just our use of them. I think the status goes with the type, not the reason.

So I read them like `<Type>? <Status>, because <Reason>. Extra info: <Message>`

So in this case, "ClusterServiceVersionCompliant? False, because RelevantCSVNotFound. Extra info: A relevant installed ClusterServiceVersion could not be found"

Reply:
Right, that makes sense.